setup.examples/exim/etc/exim4/conf.d/00_mailscanner_listmacrosdefs

```diff
@@ -0,0 +1,82 @@
+# General config for MailScanner Gateway
```

Comment:
Error, the file should be in etc/exim4/conf.d/main/

New version for conf.d/main/00_mailscanner_listmacrosdefs
I reduced options to the minimum and added some comments.

```
# General Exim config for MailScanner Gateway
# conf.d/main/00_mailscanner_listmacrosdefs

# Use these specific config files:
# conf.d/main/01_mailscanner_config
# mailscanner_acldefs

# Exim deamon listening port
daemon_smtp_ports = 25 : 587

# Activate TLS/SSL configuration for Exim 
# exim.crt and exim.key files needed before activating the feature.
#MAIN_TLS_ENABLE = Yes

# Do a reverse DNS lookup on all incoming IP calls
MAIN_HOST_LOOKUP = Yes

# Access list of domains to relay
# Domains to route are in /etc/exim4/hubbed_hosts 
MAIN_RELAY_TO_DOMAINS = /etc/exim4/relay_domains

# To activate sender callback on remote host (mailhub)
# Useful to avoid bad bounce (do verify at SMTP time)
CHECK_RCPT_LOCAL_ACL_FILE = CONFDIR/mailscanner_acldefs
#RECIPIENT_VERIFY_CALLBACK_RELAY = true

# Various Debian's ACL check
CHECK_MAIL_HELO_ISSUED = true
CHECK_RCPT_VERIFY_SENDER = true
CHECK_RCPT_LOCAL_LOCALPARTS = true
CHECK_RCPT_REMOTE_LOCALPARTS = true
# Warn if the sender host does not have valid reverse DNS.
# If you need to reject mail, you need to add your own ACL
# and comment out this one.
CHECK_RCPT_REVERSE_DNS = true

# Various Debian's variables
MAIN_TIMEOUT_FROZEN_AFTER = 2d
MAIN_FREEZE_TELL = postmaster
MESSAGE_SIZE_LIMIT = 50M
MAIN_IGNORE_BOUNCE_ERRORS_AFTER = 3h

# Various Exim variables
rfc1413_hosts = *
rfc1413_query_timeout = 0s
smtp_banner = $smtp_active_hostname ESMTP $tod_full

# To activate logging
MAIN_LOG_SELECTOR = +address_rewrite \
+all_parents \
-arguments \
+connection_reject \
+delay_delivery \
+delivery_size \
+dnslist_defer \
+incoming_interface \
+incoming_port \
+lost_incoming_connection \
+queue_run \
+received_sender \
+received_recipients \
-retry_defer \
+sender_on_delivery \
+size_reject \
-skip_delivery \
+smtp_confirmation \
+smtp_connection \
+smtp_protocol_error \
+subject \
+tls_cipher \
+tls_peerdn
```